### PR TITLE
Adding a way to list packages tracked by PY3PATCH-REQUESTED

### PR DESCRIPTION
**Pull request by torsava**
So far I've only added a TrackingBug(s) class/table into the DB scheme.

### CODE REVIEW

**Comment by encukou** [comment]
Looks reasonable, but please add the change to the DNF plugin and the load code as well

**Comment by torsava** [comment]
Of course!

File: dnf-plugins/py3query.py [hunk 1]
```diff
@@ -24,13 +24,21 @@
...
 
 BUGZILLA_URL = 'bugzilla.redhat.com'
+# Tracker bugs which are used to find all relevant package bugs
 TRACKER_BUG_IDS = [
     1285816,  # The Python 3 tracking bug
-    1322027   # The Python 3 Upstream Porting tracking bug
+    1322027,  # The Python 3 Upstream Porting tracking bug
 ]
-# Trackers of bugs that indicate the given package is mispackaged.
+# Trackers of bugs wose presence indicates the given package is mispackaged.
```
Comment (encukou) [inline dnf-plugins/py3query.py]: `whose`

File: portingdb/tables.py [hunk 2]
```diff
@@ -267,6 +267,27 @@ def __repr__(self):
                                        self.collection_package.name)
 
 
+class TrackingBug(TableBase):
+    u"""Tracking bugs associated with a package."""
+    __tablename__ = 'tracking_bugs'
+    __table_args__ = {'sqlite_autoincrement': True}
```
Comment (encukou) [inline portingdb/tables.py]: I don't think autoincrement is necessary. But please add a unique constraint.

File: portingdb/tables.py [hunk 3]
```diff
@@ -267,6 +267,27 @@ def __repr__(self):
...
+
+    id = IDColumn()
+    collection_package_id = Column(
+        ForeignKey(CollectionPackage.id), nullable=False)
+    tracking_bug = Column(
+        Integer(), nullable=False,
+        doc='ID number of the tracking bug')
+
+    collection_package = relationship(
+        'CollectionPackage', backref=backref('tracking_bugs'))
```
Comment (encukou) [inline portingdb/tables.py]: Look at at the bottom of tables.py for a few examples of doing an N-to-M relationship with intermediate table.

**Comment by torsava** [comment]
I think the pull request is ready.

**Comment by encukou** [comment]
Looks good, thanks!

**Comment by torsava** [comment]
My pleasure!

**Pull request closed without merging**